modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics using FlowMetrics API
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
```

Comment:
I am not sure how the UI looks like. You could call it something like `dropdown`, `section` or the UI element that you are pointing to instead of `heading`.
```suggestion
. In the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
```